src/MusicalScore/MusicParts/MusicPartManagerIterator.ts

```diff
@@ -565,5 +565,36 @@ export class MusicPartManagerIterator {
         return entries;
     }
 
+    /**
+     * caclculate timestemp at current cursor
+     * @param {Fraction} rhythmInstruction
+     * @returns {number}
+     */
+    public nextVoiceEntryTimeStemp(rhythmInstruction: Fraction): number {
```

Comment:
function is called nextVoiceEntryTimeStemp but returns a number or undefined. Is that on purpose?